src/io/bed.rs

```diff
@@ -273,9 +273,9 @@ mod tests {
         let starts = [5, 3];
         let ends = [5000, 5005];
 
-        let mut reader = Reader::new(BED_FILE);
```

Comment:
@pmarks in your previous PR, you committed a test case for compact bed files. However, you accidentally used the normal bed file as input. When fixing the test in this PR, I discovered that it actually does not pass. Could you have a look what is wrong? Apparently, the strategy using `Option` for aux does not seem to work.

Reply:
looking at it now...

Reply:
This is going to be non-trivial to fix & I don't think I can do it immediately.  So we should probably disable the test & put in a TODO for now.